src/controllers/AdminController.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-restricted-globals */
+
+import database from '../database/models';
+import UserService from '../services/UserServices';
+import getRoleName from '../utils/GetRoleUtil';
+
+const { findUserById } = UserService;
+/**
+ * @class UserController
+ */
+class AdminController {
+/**
+   * @param {Object} req object
+   * @param {Object} res object
+   * @returns {Object} res
+   */
+  static async changeRole(req, res) {
+    const id = Number(req.params.id);
+    if (isNaN(id)) return res.status(400).send({ status: 400, error: 'Bad request', message: `ID param value '${req.params.id}' should be an integer` });
+    try {
+      const foundUser = await findUserById(id);
+      if (!foundUser) return res.status(404).send({ status: 404, error: `User with id ${req.params.id} does not exist` });
+      const oldRole = foundUser.role_value;
+      if (oldRole === req.body.new_role) return res.status(409).send({ status: 409, message: 'User already has the role', errpr: 'conflict' });
+      await database.User.update(
```

Comment:
Personally I think it's better having this in Services for consistency

Reply:
how do you suggest I go about it?

Reply:
I suggest you add it to the `UserServices` and create a method for doing those functionalities there.
PS: that's just my opinion as longs as it works its okay for me